src/components/Dropdown.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import './dropdown.css';
+
+class Dropdown extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {display: 'Home'};
+        this.handleOption = this.handleOption.bind(this);
+    };
+
+    handleOption = (e) => {
+        e.preventDefault();
+        this.state.display.setState(e.target.display);
```

Comment:
aici asa se pare ca ceva nu este in regula

`this.setState({ display: e.target.display })` <-- asa este the correct way